test/cmdlineTests.sh

```diff
@@ -272,6 +275,9 @@ SOLTMPDIR=$(mktemp -d)
             continue
         fi
         echo "$f"
+
+        solhint -f table "$SOLTMPDIR/$f"
```

Comment:
This could be quite annoying with every change to the syntax we do. Can we somehow limit the errors in that area?